### PR TITLE
Consolidation

### PR DESCRIPTION
**Pull request by kaymckelly**
renaming and merging redundant css

### CODE REVIEW

File: v4/header.html [hunk 1]
```diff
@@ -15,7 +15,7 @@ <h1 class="d-md-none">IACR</h1>
   </div>
 </div>
 
-<div id="menubar" class="container-fluid default-menubar">
+<div id="navbar" class="container-fluid">
```
Comment (kmccurley) [inline v4/header.html]: navbar seems like a bad choice for this id.

File: v4/iacr.css [hunk 2]
```diff
@@ -23,10 +23,11 @@
 
 
 /*** Navbar (expanded) ***/
-#iacrLogo {
+/* TODO: this doesn't appear to do anything; checking if other instances exist/actually do a thing */
+/* #iacrLogo {
   max-height: 120px;
   max-width: 120px;
-}
+} */
```
Comment (kmccurley) [inline v4/iacr.css]: We need to protect ourselves against changes to the logo that might result in a bigger file. I suggest leaving it here.

File: v4/iacr.css [hunk 3]
```diff
@@ -84,22 +85,22 @@
...
-img.searchBtn {
+img.searchIcon {
   height: 20px;
 }
 
 .searchButtons {
   background-color: transparent;
-  border-color: #ffffff;
-  color: #ffffff;
+  color: #F0F0F0;
```
Comment (kmccurley) [inline v4/iacr.css]: I assume this is to make things consistent in the menu?

File: v4/iacr.css [hunk 4]
```diff
@@ -168,13 +169,14 @@ searchDropdown2 is sized correctly at different widths.  */
...
 }
+/* END searchDropdown2 sizing specs */
 
 
 
 /*** Footer ***/
 @media only screen and (min-width: 767px) {
   footer#v1 {
-    font-size: 80%;
+    font-size: 90%;
```
Comment (kmccurley) [inline v4/iacr.css]: I'd like to see the result of this change.

File: v4/themes.css [hunk 5]
```diff
@@ -1,46 +1,39 @@
...
 
 /*** Navbar ***/
-.navbar-iacr-acronym {
-  color: #ffffff !important;
-}
-
 /* TODO: build custom bootstrap for these */
 a.nav-link, a.dropdown-item {
-  color: #EBF7EB !important;
+  color: #F0F0F0 !important;
```
Comment (kmccurley) [inline v4/themes.css]: Is this to improve contrast? What is the effect?

File: v4/header.html (hunk 1 above)
Comment (kaymckelly) [reply v4/header.html]: Resolved (see commit 401761d1e55008c33404576aeb70b5bd1c07b609)

File: v4/iacr.css (hunk 2 above)
Comment (kaymckelly) [reply v4/iacr.css]: Uncommented and todo remove (see commit 774533f6196b543d48f9e3546c3dc6298253475b)

File: v4/iacr.css (hunk 3 above)
Comment (kaymckelly) [reply v4/iacr.css]: Yes

File: v4/iacr.css (hunk 4 above)
Comment (kmccurley) [reply v4/iacr.css]: Now I see it. Looks better.

**Pull request merged**